static/js/activity.js

```diff
@@ -520,13 +520,25 @@ exports.searching = function () {
     return $('.user-list-filter').expectOne().is(':focus');
 };
 
+function update_buttons_with_focus() {
```

Comment:
Can we come up with a better name for this function?  Since it's really only touching the search button, perhaps call it `update_search_button()`?